src/odemis/driver/autoscript_client.py

```diff
@@ -1454,6 +1463,12 @@ def __init__(self, name: str, role: str, parent: SEM, channel: str, **kwargs):
         # this makes it pretty annoying to do anything on that side, and error prone.
         # disabling this until a better solution is found
 
+        # median filter applied to the image (required for cryo data)
+        self.median_filter = model.FloatContinuous(
+            value=self.getMetadata().get(model.MD_MEDIAN_FILTER, 3),
+            range=(0, 9),
+        )
```

Comment:
I would have it turned off by default. We will most likely also use this code for non-cryo data.
Also I think it should be an `int` not a `float`, since you are using it as the size input of the median filter function.